frontend/src/components/forms/custom-text/CustomText.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {TextField} from "@mui/material";
+import {ITextFieldProps} from "../../../interfaces/INewItem";
+
+//component imports
```

Comment:
if you don't import something you could leave them out :-) But in general nice structure for the imports

Reply:
they get autogenerated by the script I use to create my component files. I actually forgot to move the imports to the right location in this file though, I notice now ;-)